pwiz_tools/Skyline/Controls/SeqNode/PeptideTreeNode.cs

```diff
@@ -118,39 +118,30 @@ private static int GetTypeImageIndex(PeptideDocNode nodePep)
             {
                 return (int) (nodePep.HasLibInfo
                                   ? SequenceTree.ImageId.peptide_lib_decoy
-                                  : SequenceTree.ImageId.peptide_decoy);
-            }
-            if (!nodePep.IsProteomic)
-            {
-                if (nodePep.GlobalStandardType == StandardType.GLOBAL_STANDARD ||
-                    nodePep.GlobalStandardType == StandardType.SURROGATE_STANDARD)
-                {
-                    return (int) SequenceTree.ImageId.molecule_standard;
-                }
-                return (int)SequenceTree.ImageId.molecule;
+                                  : SequenceTree.ImageId.peptide_decoy); // Decoy has no small-molecule equivalent 
             }
             if (nodePep.GlobalStandardType == StandardType.IRT)
             {
                 return (int) (nodePep.HasLibInfo
-                                  ? SequenceTree.ImageId.peptide_irt_lib
-                                  : SequenceTree.ImageId.peptide_irt);
+                                  ? (nodePep.IsProteomic ? SequenceTree.ImageId.peptide_irt_lib : SequenceTree.ImageId.molecule_irt_lib)

```

Comment:
Feels like this is getting a bit complicated. I wonder if there might be some way to have two parallel arrays for proteomic and non-proteomic, pick the right array at the top and then just index into it by another enum to get the right value.

Reply:
I've created a small dictionary for images that have peptide/molecule versions.